packages/Search/test/tstBoostRangeAdapters.cpp

```diff
@@ -0,0 +1,37 @@
+/****************************************************************************
+ * Copyright (c) 2012-2018 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <DTK_DetailsUtils.hpp> // iota
+
+#include <Teuchos_UnitTestHarness.hpp>
+
+#include <boost/range/algorithm.hpp> // reverse_copy, replace_if, count
+#include <boost/range/numeric.hpp>   // accumulate
+
+#include "DTK_BoostRangeAdapters.hpp"
+
+#include <sstream>
+
+TEUCHOS_UNIT_TEST( BoostGeometryAdapters, Range )
+{
+    Kokkos::View<int[4], Kokkos::HostSpace> w( "w" );
```

Comment:
Can you add a test where you work with something else than a POD?

Reply:
I amended changes.   Computing an approximation to pi :)